### ISSUE TITLE
reverse signal option during compilation in web builder

### ISSUE THREAD

**Issue by FusionHolic** [grblHAL/STM32F1xx#61]
halo terje, apakah ada opsi untuk membalikkan sinyal input menjadi no saat kompilasi di web builder, karena dengan default nc cukup membingungkan bagi pemula saat driver alarm dn tidak bis digunakan di grbl sender yang umum seperti openbuilds atau ugs

**Comment by terjeio** [comment]
No, not yet. It is on my longer term plan to add machine profiles to the Web Builder that will allow it.
